Scripts/install_insar_analysis_pkgs.sh

```diff
@@ -3,17 +3,14 @@ set -e
 env="insar_analysis"
 local="/home/jovyan/.local"
 env_prefix=$local"/envs/"$env
-site_packages=$env_prefix"/lib/python3.8/site-packages"
+site_packages=$env_prefix"/lib/python3.11/site-packages"
```

Comment:
It seems like this is going to break the next time that a major version of python is released -- how do we explicitly take the option we want (which I think would be the last item in a sorted list of "python*" directories)?

Reply:
Good thought. I think I'll do it with `python_version=$(conda run -n $env python --version | cut -b 8-11)`